app/controllers/account_invitations_controller.rb

```diff
@@ -2,16 +2,16 @@ class AccountInvitationsController < ApplicationController
   before_action :authenticate_user!
 
   def index
-    @account_invitations = current_user.account_invitations
+    @account_invitations = account.invitations
   end
 
   def new
     @account_invitation = AccountInvitation.new
   end
 
   def create
-    AccountInvitation.create!(user_id: current_user.id, **account_invitation_params)
-    redirect_to account_invitations_path
+    AccountInvitation.create!(user_id: current_user.id, account_id: ps.fetch(:account_id), **account_invitation_params)
```

Comment:
there is a security issue here - it doesn't if the current user can do this operation with the given account_id.